packages/documentation/src/content/docs/integration/requirements/peers.mdx

```diff
@@ -0,0 +1,287 @@
+    message
+    peer {
+      id
+      asset {
+        code
+        scale
+      }
+      staticIlpAddress
+      name
+    }
+  }
+}
+```
+
+</CodeBlock>
+
+##### Example
+
+<CodeBlock title='Example JSON request'>
+
+```json
+{
+  "input": {
+	"staticIlpAddress": "g.othergreatwallet",
+	"name": "The Other Great Wallet"
+	"http": {
+  		"incoming": {"authTokens": ["mytoken"]},
+  		"outgoing": {"endpoint": "ilp.othergreatwallet.com", "authToken": "theirtoken"}
+	  },
+	"assetId": "INSERT_ASSET_ID",
+	"initialLiquidity": <optionally, an intial amount of liquity to provision. Liquidity can also be deposited via the `DepositPeerLiquidity`>
+  }
+}
+```
+
+</CodeBlock>
+
+<div class="overflow-table">
+
+| Variable                   | Description                                                                                                                 | Required |
+| -------------------------- | --------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `assetID`                  | The ID of the asset that you and your peer will use to ultimately settle your net obligations outside of Interledger.       | Y        |
+| `staticILPaddress`         | Your peer’s static ILP address (the example uses `g.othergreatwallet`)                                                      | Y        |
+| `name`                     | Your peer’s name (the example uses “The Other Great Wallet”)                                                                | Y        |
+| `http.incoming.authTokens` | The token your peer will use to present and connect to and send packets to your Rafiki instance.                            | Y        |
+| `http.outgoing.endpoint`   | Your peer’s connector endpoint. By default it is on local port 0.0.0.0:3002                                                 | Y        |
+| `http.outgoing.authtoken`  | The token that you will use to present to your peer and connect to and send packets to their Rafiki instance.               | Y        |
+| `initialLiquidity`         | Initial amount of liquidity to deposit for peer. Liquidity can also be deposited using the `DepositPeerLiquidity` mutation. | N        |
+
+</div>
+
+<CodeBlock title="Example JSON response">
+
+```json
+{
+  "data": {
+    "createPeer": {
+      "code": "200",
+      "success": true,
+      "message": "Created ILP Peer",
+      "peer": {
+        "id": "480ef339-7842-4501-a905-923fc1339cef",
+        "asset": {
+          "code": "USD",
+          "scale": 2
+        },
+        "staticIlpAddress": "g.othergreatwallet",
+        "name": "The Other Great Wallet"
+      }
+    }
+  }
+}
+```
+
+</CodeBlock>
+
+### Create wallet addresses
+
+After adding a peer, we need to create a wallet address. This wallet address is where payments are sent from and received. To learn how to create a new wallet address, refer to [Assets](/integration/requirements/wallet-addresses/).
+
+## Managing peers
+
```

Comment:
Let's also talk about this section. It makes it sound like the ASE should expect to perform ongoing maintenance, but in an ideal world, I don't believe there should be much for the ASE to worry about other than liquidity. I have a few more thoughts but it's a lot to type out.

Reply:
I reworded the introductory paragraph to deemphasize the frequency of editing peers. Hopefully this change captures the essence of "you shouldn't have to edit these peers once they are set up properly". I also moved the contents of "Edit a peer" and "Delete a peer" into details elements to lessen the screen real estate impact that made it feel like they were a significant part of the workflow.